client/src/app/videos/recommendations/recent-videos-recommendation.service.spec.ts

```diff
@@ -16,47 +17,61 @@ describe('"Recent Videos" Recommender', () => {
       service = new RecentVideosRecommendationService(videosService)
```

Comment:
May want to give this a different name now that the behaviour is changing.  Maybe "RecentTaggedVideosService".  (Can probably drop the "Recommendation" for the sake of brevity, and because it's implied by the interface)